src/pds_doi_service/core/actions/test/release_test.py

```diff
@@ -79,14 +79,15 @@ def test_reserve_release_to_review(self):
             'no_review': False
         }
 
-        o_doi_labels = self._action.run(**release_args)
+        o_doi_label = self._action.run(**release_args)
 
-        for o_doi_label in o_doi_labels:
-            dois, _ = DOIOstiJsonWebParser.parse_dois_from_label(o_doi_label)
+        dois, _ = DOIOstiJsonWebParser.parse_dois_from_label(o_doi_label)
 
-            # Should get one DOI back that has been marked as ready for review
-            self.assertEqual(len(dois), 1)
-            self.assertTrue(dois[0].status == DoiStatus.Review)
+        # Should get one DOI back that has been marked as ready for review
+        self.assertEqual(len(dois), 1)
+
+        for doi in dois:
+            self.assertTrue(doi.status == DoiStatus.Review)
```

Comment:
Maybe consider using the extra argument in the `TestCase.assert*` methods that let you document what went wrong, e.g.:
```python
self.assertTrue(doi.status == DoiStatus.Review, f'{Doi {doi} had status {doi.status} and not expected DoiStatus.Review')
```

Then when you run the tests and it fails, you don't just get `False is not True` but `False is not True: Doi 12345 had status <DoiStatus: Published> and not expected DoiStatus.Review`.

👉 Another idea: use `assertEqual`